docs/en/kubernetes/Metrics-On-Kubernetes.md

```diff
@@ -0,0 +1,300 @@
+
+This documentation focus on how to configure and get metrics of different metrics sinks from Alluxio deployed on Kubernetes.
+
+* Table of Contents
+{:toc}
+
+### HTTP JSON Sink
+
+#### Configuration
+
+The metrics are exposed through the web ports of different components.
+
+* Alluxio masters and workers web ports are opened by default.
+* Alluxio standalone Fuse web port is not opened by default. It can be opened by setting `alluxio.fuse.web.enabled` to true.
+
+#### Get Metrics Snapshot
+
+You can send an HTTP request to an Alluxio process to get a snapshot of the metrics in JSON format.
+```console
+# Get the metrics in JSON format from one specific component
+$ kubectl exec <COMPONENT_HOSTNAME> -c <CONTAINER_NAME> -- curl 127.0.0.1:<COMPONENT_WEB_PORT>/metrics/json/
+
+# For example, get the metrics of the leading master with default web port 19999
+$ kubectl exec <alluxio-master-x> -c alluxio-master -- curl 127.0.0.1:19999/metrics/json/
+# Get the metrics of a worker with default web port 30000
+$ kubectl exec <alluxio-worker-xxxxx> -c alluxio-worker -- curl 127.0.0.1:30000/metrics/json/
+# Get the metrics of the leading job master with default web port 20002
+$ kubectl exec <alluxio-master-x> -c alluxio-job-master -- curl 127.0.0.1:20002/metrics/json/
+# Get metrics of a job worker with default web port 30003
+$ kubectl exec <alluxio-worker-xxxxx> -c alluxio-job-worker -- curl 127.0.0.1:30003/metrics/json/
+# Get metrics of a fuse process with default web port 49999
+$ kubectl exec <alluxio-fuse-xxxxx> -- curl 127.0.0.1:49999/metrics/json/
+```
+
+#### Visualization
+
+##### Master Web UI Metrics
+
+Besides the raw metrics shown via metrics servlet or custom metrics configuration,
+users can track key cluster performance metrics in a more human-readable way in the web interface of
+Alluxio leading master.
+
+![Master Metrics]({{ '/img/screenshot_generalMetrics.png' | relativize_url }})
+
+To access the wbe UI of the leading master, you need to forward a port on a physical machine to the master web port.
+For example, if your leading master is running in pod `alluxio-master-0` and you want to serve the Alluxio master web UI at 
+node `master-node-1` on port `8080`, you can run the following command on your control plane:
+```console
+$ kubectl port-forward <CONTROL_PLANE_HOSTNAME> pods/alluxio-master-0 8080:19999
+```
+
+The metrics can be then accessed at `http://<leading_master_node_hostname>:<forwared_port>/metrics`
+
+Refer to [Access the Web UI]({{ '/en/kubernetes/Running-Alluxio-On-Kubernetes.html' | relativize_url }}#access-the-web-ui) for more detailed instructions.
+
+
+### Prometheus Sink
+
+[Prometheus](https://prometheus.io/) is a monitoring tool that can help to monitor Alluxio metrics changes.
+
+#### Alluxio Helm Chart Configuration
+`PrometheusMetricsServlet` needs to be enabled for Prometheus in Alluxio. In the helm chart `values.yaml` file, set the following properties:
```

Comment:
Sets the following properties in helm chart `values.yaml` to enable the Prometheus metrics sink